modules/lib/lib-project/src/test/resources/test/ModifyProjectReadAccessHandlerTest.js

```diff
@@ -0,0 +1,56 @@
+var assert = require('/lib/xp/testing.js');
+var project = require('/lib/xp/project.js');
+
+exports.modifyReadAccess = function () {
+    createProject();
+
+    var result = project.modifyReadAccess({
+        name: 'myproject',
+        readAccess: {public: false}
+    });
+
+    assert.assertJsonEquals({readAccess: {public: false}}, result);
+
+    result = project.modifyReadAccess({
+        name: 'myproject',
+        readAccess: {public: true}
+    });
+
+    assert.assertJsonEquals({readAccess: {public: true}}, result);
+};
+
+exports.modifyReadAccessNull = function () {
+    createProject();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createProject' was used before it was defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5511138)